_targets.R

```diff
@@ -4,44 +4,69 @@ library(tibble)
 library(dplyr)
 
 options(tidyverse.quiet = TRUE)
-tar_option_set(packages = c("tidyverse", "dataRetrieval", "urbnmapr", "rnaturalearth", "cowplot", "lubridate"))
+tar_option_set(packages = c("tidyverse", "dataRetrieval", "urbnmapr", "rnaturalearth", "cowplot", "lubridate", "leaflet", "leafpop", "htmlwidgets"))
 
 # Load functions needed by targets below
 source("1_fetch/src/find_oldest_sites.R")
 source("1_fetch/src/get_site_data.R")
 source("3_visualize/src/map_sites.R")
 source("2_process/src/tally_site_obs.R")
 source("3_visualize/src/plot_site_data.R")
+source("3_visualize/src/plot_data_coverage.R")
+source("2_process/src/summarize_targets.R")
+source("3_visualize/src/map_timeseries.R")
 
 # Configuration
 states <- c('WI','MN','MI', 'IL', 'IN', 'IA')
 parameter <- c('00060')
 
 # Targets
+
+# Static branching with splitting, appliers, and now combiners (this is why it's outside of the list)
+mapped_by_state_targets <- tar_map(
+  values = tibble(state_abb = states)%>%
+    mutate(state_plot_files = sprintf("3_visualize/out/timeseries_%s.png", state_abb)),
+  tar_target(nwis_inventory, get_state_inventory(sites_info = oldest_active_sites, state_abb)),
+  tar_target(nwis_data, get_site_data(nwis_inventory, state_abb, parameter)),
+  tar_target(tally, tally_site_obs(nwis_data)),
+  tar_target(timeseries_png, plot_site_data(state_plot_files, nwis_data, parameter), format = "file"),
+  names = state_abb,
+  unlist = FALSE
+)
+
 list(
   # Identify oldest sites
   tar_target(oldest_active_sites, find_oldest_sites(states, parameter)),
 
-  # # TODO: PULL SITE DATA HERE
-  # tar_target(wi_data, get_site_data(oldest_active_sites, states[1], parameter)),
-  # tar_target(mn_data, get_site_data(oldest_active_sites, states[2], parameter)),
-  # tar_target(mi_data, get_site_data(oldest_active_sites, states[3], parameter)),
-
-  # Static branching with splitting and appliers
-  tar_map(
-    values = tibble(state_abb = states)%>%
-      mutate(state_plot_files = sprintf("3_visualize/out/timeseries_%s.png", state_abb)),
-    tar_target(nwis_inventory, get_state_inventory(sites_info = oldest_active_sites, state_abb)),
-    tar_target(nwis_data, get_site_data(nwis_inventory, state_abb, parameter)),
-    tar_target(tally, tally_site_obs(nwis_data)),
-    tar_target(timeseries_png, plot_site_data(state_plot_files, nwis_data, parameter), format = "file"),
-    names = state_abb
-    ),
+  # defined above
+  mapped_by_state_targets,
+
+  # combine tally_[state], the third target in tar_map
+  tar_combine(obs_tallies, mapped_by_state_targets[3], command = combine_obs_tallies(!!!.x)),
```

Comment:
The other way to reference specific parts of mapped output is to use the target name. I think it can make it clearer as you read through in the future, plus prevent future issues if you ever change the order or add a new step thereby changing the index that goes to `tally`. So instead of `mapped_by_state_targets[3]`, you could write as `mapped_by_state_targets$tally`.

Reply:
I like this. Fewer things to make mistakes on.